pages/mesosphere/dcos/1.14/installing/production/deploying-dcos/installation/index.md

```diff
@@ -415,21 +429,15 @@ At this point your directory structure should resemble:
 
     __Note:__ If you encounter errors such as `Time is marked as bad`, `adjtimex`, or `Time not in sync` in journald, verify that Network Time Protocol (NTP) is enabled on all nodes. For more information, see the [system requirements](/mesosphere/dcos/1.14/installing/production/system-requirements/ports/) documentation.
 
-5.  Monitor Exhibitor and wait for it to converge at `http://<master-ip>:8181/exhibitor/v1/ui/index.html`.
-
-    <p class="message--note"><strong>NOTE: </strong>This process can take about 10 minutes. During this time, you will see the Master nodes become visible on the Exhibitor consoles and come online, eventually showing a green light.</p>
-
-![Exhibitor for ZooKeeper](/mesosphere/dcos/1.14/img/chef-zk-status.png)
+5.  Monitor the DC/OS web interface and wait for it to converge at at: `http://<master-node-public-ip>/`. 
```

Comment:
The UI does not converge, it either displays or it does not :-).

"... and wait for the UI to be served".

Reply:
`at at`